### PR TITLE
[Enhancement][API]Enhance mysql connection properties

### PR DESCRIPTION
**Pull request by lgcareer**
## Purpose of the pull request
This pull request enhance mysql connection properties

## Brief change log

- *Enhance mysql connection properties*

### CODE REVIEW

**Comment by zhongjiajie** [comment]
tests error, could you please fix it?

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-mysql/src/main/java/org/apache/dolphinscheduler/plugin/datasource/mysql/param/MySQLDataSourceProcessor.java [hunk 1]
```diff
@@ -140,7 +132,32 @@ public Connection getConnection(ConnectionParam connectionParam) throws ClassNot
...
+                    log.info("Key `{}` is not legitimate for security reason", k);
+                    return;
+                }
+                connectionProperties.put(k, v);
+            });
+        }
+        connectionProperties.put(AUTO_DESERIALIZE, "false");
+        connectionProperties.put(ALLOW_LOAD_LOCAL_IN_FILE_NAME, "false");
+        connectionProperties.put(ALLOW_LOCAL_IN_FILE_NAME, "false");
+        connectionProperties.put(ALLOW_URL_IN_LOCAL_IN_FILE_NAME, "false");
```
Comment (qingwli) [inline dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-mysql/src/main/java/org/apache/dolphinscheduler/plugin/datasource/mysql/param/MySQLDataSourceProcessor.java:159]: https://github.com/apache/dolphinscheduler/pull/15430#discussion_r1442447272

**Comment by lgcareer** [comment]
> tests error, could you please fix it?

Yes,has fixed.

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-mysql/src/main/java/org/apache/dolphinscheduler/plugin/datasource/mysql/param/MySQLDataSourceProcessor.java (hunk 1 above)
Comment (zhongjiajie) [reply dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-mysql/src/main/java/org/apache/dolphinscheduler/plugin/datasource/mysql/param/MySQLDataSourceProcessor.java:159]: Seem L150 already add log for it

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Comment by HarshitNagpal29** [comment]
> LGTM, thanks

@zhongjiajie sir, sorry to disturb you but i am waiting for your approval and response in HTTP task parameter issue also I have tried to contact you via email.

**Pull request merged**